File: templates/Uwp/_composition/CodeBehind/Project.SplitView.AddToSecondaryNavigationItems/Views/ShellPage.xaml_postaction.cs
//{**
//This code block adds the wts.ItemNamePage to the _secondaryItems of the ShellPage.
//**}

namespace Param_ItemNamespace.Views
{
    public sealed partial class ShellPage : Page, INotifyPropertyChanged
    {
        private void PopulateNavItems()
        {
            //^^
            //{[{
            _secondaryItems.Add(ShellNavigationItem.FromType<wts.ItemNamePage>("Shell_wts.ItemName".GetLocalized(), Symbol.Setting));
            //}]}
        }
    }
}

File: templates/Uwp/_composition/CodeBehind/Project.SplitView.AddToSecondaryNavigationItemsVB/Views/ShellPage.xaml_postaction.vb
'{**
'This code block adds the wts.ItemNamePage to the _secondaryItems of the ShellPage.
'**}

Namespace Views
    Public NotInheritable Partial Class ShellPage
        Inherits Page
        Implements INotifyPropertyChanged

        Private Sub PopulateNavItems()
            '^^
            '{[{
            _secondaryItems.Add(ShellNavigationItem.FromType(Of wts.ItemNamePage)("Shell_wts.ItemName".GetLocalized(), Symbol.Setting))
            '}]}
        End Sub
    End Class
End Namespace
